rocky/reports/report_types/dns_report/report.py

```diff
@@ -24,7 +24,9 @@ class DNSReport(Report):
 
     def generate_data(self, input_ooi: str, valid_time: datetime) -> dict[str, Any]:
         ref = Reference.from_str(input_ooi)
-        tree = self.octopoes_api_connector.get_tree(ref, valid_time, depth=3, types={DNSRecord, Finding}).store
+        tree = self.octopoes_api_connector.get_tree(
+            ref, valid_time=valid_time, depth=3, types={DNSRecord, Finding}
+        ).store
```

Comment:
This is because `get_tree` of the octopoes connector has changed to make valid_time a required parameter, but the mock octopoes connector method signature has not been updated. Instead of changing this code, the mock octopoes connector get_tree method should be changed to match the real one. E.g. to this:

```
    def get_tree(
        self, reference: Reference, valid_time: datetime, types: Set = frozenset(), depth: int = 1
    ) -> ReferenceTree:
```

`Set` is `collections.abc.Set` here

Reply:
Should be fixed now.